plugin/src/test/groovy/com/novoda/test/TestAndroidKotlinProject.groovy

```diff
@@ -14,7 +14,8 @@ buildscript {
     }
 }
 plugins {
-    ${formatPlugins(project)}
+    ${formatPlugins(project)}   
+    id 'com.novoda.static-analysis'
```

Comment:
It is better if the `static-analysis` plugin is added later. I needed to do this earlier. It may not be needed anymore. But I think it is a good change anyways since it makes it explicit. Before it was added by the method `withPlugin` in the constructor.